corehq/apps/export/models/new.py

```diff
@@ -230,42 +289,70 @@ def _generate_schema_from_xform(xform, langs, appVersion):
 
         return schema
 
-    @staticmethod
-    def _merge_schema(schema1, schema2):
-        """Merges two ExportDataSchemas together
 
-        :param schema1: The first ExportDataSchema
-        :param schema2: The second ExportDataSchema
-        :returns: The merged ExportDataSchema
-        """
+class CaseExportDataSchema(ExportDataSchema):
 
-        schema = ExportDataSchema()
+    @staticmethod
+    def generate_schema_from_builds(domain, app_id, case_type):
+        app_build_ids = get_built_app_ids_for_app_id(domain, app_id)
+        all_case_schema = CaseExportDataSchema()
 
-        def resolvefn(group_schema1, group_schema2):
-            group_schema = ExportGroupSchema(
-                path=group_schema1.path,
-                last_occurrence=max(group_schema1.last_occurrence, group_schema2.last_occurrence),
+        for app_doc in iter_docs(Application.get_db(), app_build_ids):
+            app = Application.wrap(app_doc)
+            case_type_metadata = filter(
+                lambda case_type_meta: case_type_meta.name == case_type,
+                app.get_case_metadata().case_types
+            )[0]
```

Comment:
out of curiosity what are the main differences between `get_case_metadata` and [`get_case_properties`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/util.py#L224)